prow/tide/tide_test.go

```diff
@@ -2820,6 +2842,8 @@ func TestIsPassing(t *testing.T) {
 				OptionalContexts:    []string{"c4"},
 				SkipUnknownContexts: &yes,
 			},
+			availableContexts: []string{"c1", "c2", "c3", "c4", "c5"},
+			failedContexts:    []string(nil),
```

Comment:
I think If its nil slice we can remove this line.

Reply:
Addressed!